javascript/react-js/inputs-and-lists.md

```diff
@@ -59,9 +59,10 @@ ReactDOM.render(
 );
 ~~~
 
-5. For our solution, we could style the application using CSS to make it look a little bit nicer. For those who don't know, CSS is the acronym of "Cascading Style Sheets", in short, is the code that helps us style our HTML.
-If you intend on styling the application, we highly recommend that you use your own style instead of using CSS Frameworks that are beyond the current scope. Using CSS, in general, is solely to make your application look prettier. Please refer to both [MDN](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) and [React](https://reactjs.org/docs/faq-styling.html) documentation, if you wish to know more about how to style React applications with CSS.
-If you are following along with this and do not wish to style the application, you can skip this step.
+5. We may style the application using "vanilla" CSS to make it look a little bit nicer. At this point, you should be familiar with the basics of CSS from the previous material. 
+If you intend on styling the application, we highly recommend that you use your own styling instead of using CSS frameworks that are beyond the current scope of the curriculum. Please refer to both [MDN](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) and [React](https://reactjs.org/docs/faq-styling.html) documentation if you wish to know more about how to style React applications with CSS.
+
+If you are following along with this tutorial and do not wish to style the application, you can skip this step.
```

Comment:
I pasted this in the https://www.theodinproject.com/lessons/preview to double check since the preview on here showed it a bit off. it looks like line 65 wouldn't keep the indentation since there is a new line between it. On the preview thing, I added a `tab` before the text on line 65 and it looked like it lined up correctly. (it may be correct how it is, not sure though since the md viewer is finicky.

Reply:
Thanks for catching this!

Reply:
oh you reply fast lol!

Reply:
Done!